content/angular/en/test.md

```diff
@@ -66,12 +66,14 @@ yarn add storybook-chromatic
 Import Chromatic in your `.storybook/config.js` file.
 
 ```javascript
-import { configure } from '@storybook/react';
+import {
+  configure
+} from '@storybook/angular';
 import 'storybook-chromatic/storybook-addon';
+import '../src/styles.less';
```

Comment:
This one should remain .css. The user is not actually running less, rather they're including the pre-compiled .css file included in the repo.

Reply:
Well, I changed that in both the code and in the tutorial. I've configured the cli to use LESS. It didn't really make sense to use gulp and compile less when the @angular/cli + webpack can allow use of .less files.

Reply:
sgtm thanks for the explanation 👍